cypress/integration/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -250,6 +250,26 @@ function testWorkflow(registry: string, repo: string, name: string) {
       cy.contains('button', 'Restub').click();
       cy.contains('button', 'Publish').should('be.disabled');
     });
+    it('hide and un-hide a version', () => {
+      goToTab('Versions');
+      // hide
+      cy.get('[data-cy=refreshButton]').click();
+      cy.get('[data-cy=versionRow]').last().contains('button', 'Actions').should('be.visible').click();
+      cy.contains('button', 'Edit').click();
+      cy.contains('div', 'Hidden:').within(() => {
+        cy.get('[name=checkbox]').click();
+      });
+      cy.contains('button', 'Save Changes').click();
+      cy.get('[data-cy=hidden]').should('have.length', 1);
+      // un-hide
+      cy.get('[data-cy=versionRow]').last().contains('button', 'Actions').should('be.visible').click();
```

Comment:
This is required as the first option that would be selected is the default version (which cannot be hidden). It is also for this version that I didn't  just use `cy.contains('button', 'Actions')` (as it only grabs the first match). 

It would be good to come up with a way to match the first non-default option (as the default could change, breaking this test).

Reply:
The tool test just uses `cy.contains('button', 'Actions') ` as the first occurrence is not the default. It would be good to correct this as well.

Reply:
Punting on this for now.